source/slang/core.meta.slang

```diff
@@ -1122,6 +1122,11 @@ attribute_syntax [vk_binding(binding: int, set: int = 0)]			: GLSLBindingAttribu
 __attributeTarget(DeclBase)
 attribute_syntax [gl_binding(binding: int, set: int = 0)]			: GLSLBindingAttribute;
 
+__attributeTarget(DeclBase)
```

Comment:
These `__attributeTarget(DeclBase)` modifiers could probably be narrowed to `VarDeclBase` so that the user can't put `[[vk::push_constant]]` onto a function declaration.